api/app/tests/apps/api/v1/test_filters.py

```diff
@@ -543,3 +544,62 @@ def test_contact_details_store_and_retrieve(self):
         self.assertIn('contact_details', response_data['options'])
         self.assertEqual(
             set(response_data['options']['contact_details']), set(['none', 'email']))
+
+
+class TestTypeFilter(SignalsBaseApiTestCase):
+    SIGNALS_LIST_ENDPOINT = '/signals/v1/private/signals/'
+
+    def setUp(self):
+        self.signals = {
+            'SIG': SignalFactory.create_batch(5),
+            'REQ': SignalFactory.create_batch(3),
+            'QUE': SignalFactory.create_batch(1),
+            'COM': SignalFactory.create_batch(2),
+            'MAI': SignalFactory.create_batch(4),
+        }
+
+        hours = 1
+        for type_code in self.signals.keys():
```

Comment:
nit

You could use `self.signals.items()` and avoid `self.signals[type_code]` two lines down.